changelog/2793.feature.rst

```diff
@@ -0,0 +1,2 @@
+Now able to create a `sunpy.map.Map` using an array and a `astropy.wcs.WCS` object.
+Added unit test for te same.
```

Comment:
You can remove this. ONly the first line is important  for the changelog.

Reply:
Oh I thought it would be nice to give a description of all the changes done 
I will remove it

Reply:
Done:)

Reply:
I understand, its more complete. But I find users/changelogs people care about the fix or the feature. They just assume or don't care about tests.

Reply:
Oh ok ok I will keep that in mind.